litestar/response/base.py

```diff
@@ -88,11 +87,6 @@ def __init__(
         status_code = status_code or HTTP_200_OK
         self.headers = MutableScopeHeaders()
 
-        if encoded_headers is not None:
```

Comment:
I think all of these changes ought to remove the deprecated parameter as well, not just the deprecation warning :P